C++/C++.sublime-syntax

```diff
@@ -328,6 +314,38 @@ contexts:
       push: global-modifier
     - include: late-expressions
 
+  global-block-comments:
+    # NOTE: This context overrides `comments` context, which re-uses scopes
+    # from `source.c#comments`. So keep the trailing `c` for consistency reasons!
+    - match: ^(/\*) =(\s*.*?)\s*= (\*/)$\n?
```

Comment:
I know this pattern was just copied, but this `.*?` combination looks a bit suspicious or even bad to me. Am I missing something, or isn't the `?` needed here?

Reply:
It is required to give the trailing `\s*` precedence so `meta.toc-list` only includes the text stripped by whitespace. But the capture group includes the leading whitespace, which seems wrong.